src/components/common/layout/banner.njk

```diff
@@ -57,7 +57,7 @@
           type: "button",
           classes:"cookie-hide-button",
           attributes: {
-            "aria-label": "Hide cookie banner button"
+            "aria-label": "Hide this message"
```

Comment:
this page has some issues that the text is not using translations like it should